test-data/unit/semanal-typeddict.test

```diff
@@ -0,0 +1,16 @@
+-- Semantic analysis of typed dicts
+
+[case testCanDefineTypedDictType]
+from mypy.typing import TypedDict
+Point = TypedDict('Point', {'x': int, 'y': int})
+[out]
```

Comment:
@JukkaL Hey thanks for the tips. This may be enough for me to get the tests to work.

I'll probably take another look Wednesday evening.